ipatests/test_integration/test_commands.py

```diff
@@ -1489,3 +1489,28 @@ def test_ipa_cacert_manage_prune(self):
         self.master.run_command(['date', '-s', '-15Years'])
 
         assert isrgrootx1_nick in result
+
+    def test_client_doesnot_throw_responsenotready_error(self):
+        """
+        This testcase checks that ipa command
+        doesn't throw http.client.ResponseNotReady error
+        when current users session is deleted from the cache
+        """
+        error_msg = (
+            "http.client.ResponseNotReady: Request-sent \n"
+            "ipa: ERROR: an internal error has occurred"
+        )
+        domainname = "ipa.test"
+        del_ccache = "rm -fv /run/ipa/ccaches/admin@{}".format(domainname)
+
+        # kinit as admin on ipa-client and run ipa command
+        tasks.kinit_admin(self.clients[0], raiseonerr=False)
+        self.clients[0].run_command(["ipa", "user-find"])
```

Comment:
What is the purpose of this, are you expecting something? Can use assert if that's the case.